src/dbally/iql_generator/prompt.py

```diff
@@ -71,82 +112,131 @@ def __init__(self, *, question: str, examples: List[FewShotExample] = None) -> N
 
 class IQLGenerationPromptFormat(PromptFormat):
     """
-    IQL prompt format, providing a question and filters to be used in the conversation.
+    IQL prompt format, providing a question and methods to be used in the conversation.
     """
 
     def __init__(
         self,
         *,
         question: str,
-        filters: List[ExposedFunction],
-        examples: List[FewShotExample] = None,
+        methods: List[ExposedFunction],
+        examples: Optional[List[FewShotExample]] = None,
     ) -> None:
         """
         Constructs a new IQLGenerationPromptFormat instance.
 
         Args:
             question: Question to be asked.
-            filters: List of filters exposed by the view.
+            methods: List of filters exposed by the view.
```

Comment:
I assume you changed "filters" to "methods" because this can also take aggregation methods (?) but the description stills says "filters"

Reply:
Thanks for catching, will fix this